lib/index.js

```diff
@@ -24,9 +23,7 @@ var PouchMirror = module.exports = function (localDB, remote) {
   // Clone all the functions from the localDB into the PouchMirror instance

```

Comment:
I have that in mind, so far my priority was to have the remote strategy work as it is, with minimal diversions from the previous code. Once the strategy code is stable, we can do all sorts of rework.

I actually had a question about this part of the code... as we are shimming only a subset of PouchDB methods, why do you clone all of the localDB methods inside pouch-mirror?